package.json

```diff
@@ -5,7 +5,7 @@
   "homepage": "https://github.com/angular/clutz",
   "description": "Clutz is not on npm, and technically not a node package. \n This is needed only for testing Clutz's output against the TypeScript compiler, and to install some development dependencies from npm.",
   "devDependencies": {
-    "clang-format": "^1.2.3",
+    "clang-format": "^1.2.4",
```

Comment:
Unfortunately, even with this there is a difference between internal and external formatting which causes this to still fail :(

Reply:
From https://circleci.com/gh/angular/clutz/569?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```

      diff:
          @@ -39,9 +39,8 @@
           class B {
             twoLines: {param1: string|undefined} = {param1: 'param1'};
             multipleLines: {param2: string|undefined} = {param2: 'param2'};
          -  twoParams:
          -      {param3: string|undefined,
          -       param4: string|undefined} = {param3: 'param3', param4: 'param4'};
          +  twoParams: {param3: string|undefined,
          +              param4: string|undefined} = {param3: 'param3', param4: 'param4'};
           
```

Maybe  @mprobst should take a look.

Reply:
I'll need to do a clang-format release. That'll take a while as I need to get hold of a windows computer...

Reply:
No rush here, just let me know when done.

Reply:
@rkirov there's clang-format 1.3.0 on npm now.

Reply:
Great, thanks!

Unfortunately, also your latest change ended up using a "too-fresh" API from guava (?) - 

https://circleci.com/gh/angular/clutz/593?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
/home/circleci/clutz/src/main/java/com/google/javascript/clutz/DeclarationGenerator.java:1327: error: cannot find symbol
        .splitToStream(namespace)
```

So I still can't unbreak the build